sdk/eventhub/Azure.Messaging.EventHubs/tests/EventHubProducer/EventHubProducerLiveTests.cs

```diff
@@ -181,5 +181,302 @@ await using (var producer = client.CreateProducer())
                 }
             }
         }
+
```

Comment:
I don't see the test case that we thought may indicate a problem... the one with multiple producers where one was pointed to an invalid partition and may have caused sending to fail for the others.   I'd love to be able to copy/paste that and kick the tires later this afternoon or early tomorrow to see what the deal is.

Reply:
I believe you can use the the `ConsumerCannotReceiveFromInvalidPartition` for that. You'll see the test is failing because three of the test cases throw an unexpected `OperationCanceledException`. I believe both issues are triggered for the same reason. I'll include a test to make sure we can still receive from valid partitions after attempting to receive from an invalid one.

Reply:
This issue happens when receiving. I believe you can use the the `ConsumerCannotReceiveFromInvalidPartition` for that. You'll see the test is failing because three of the test cases throw an unexpected `OperationCanceledException`. I believe both issues are triggered for the same reason. I'll include a test to make sure we can still receive from valid partitions after attempting to receive from an invalid one. Maybe I should do the same for producers to make sure it works.

Reply:
I have included three tests in Consumer Live Tests to cover that. `FailingToCreateInvalidPartitionConsumerDoesNotCompromiseReceiveBehavior` is one of them. It seems that's a problem from Track One.

Reply:
Thanks!   Do you mind tagging those that were also an issue in track one with `[Ignore]` and a message indicating that they need investigation?

[Ignore docs](https://github.com/nunit/docs/wiki/Ignore-Attribute)

Reply:
I'll do that in Consumer. We have no issues in Producer.